drake/doc/no_push_to_origin.rst

```diff
@@ -0,0 +1,39 @@
+*****************************************
+Avoid Accidental Pushes to the Repository
+*****************************************
+
+The Issue
+=========
+
+We want to prevent developers from accidentally pushing changes into RobotLocomotion/drake. This is accomplished by:
+
+   1. Everybody agreeing in a naming convention for the respositories.
+   2. Providing a dummy url for the repository we want to protect.
```

Comment:
this is not really the central idea.  the core idea (which i didn't know about) is that git can set the "push" target of a remote to be different than the pull target.